lib/assets/javascripts/twine.js.coffee

```diff
@@ -268,10 +268,19 @@ Twine.bindingTypes =
   'bind-class': (node, context, definition) ->
     fn = wrapFunctionString(definition, '$context,$root', node)
     lastValue = {}
+    additions = []
+    removals = []
```

Comment:
Since `$(node)` is used several times, may as well store it at the start in a variable (e.g. `$node`).